api/controllers/user.controller.js

```diff
@@ -0,0 +1,43 @@
+import user from '../modules/user';
+
+class UserController {
+    get(req, res) {
+        const { id } = req.params;
+        const result = user.get(id);
+
+        if (!result) {
+            return res.status(404).end('User not found!');
+        }
+
+        res.json(result);
+    }
+
+    add(req, res) {
+        const { body } = req;
+
+        res.json(user.add(body));
+    }
+
+    update(req, res) {
+        const { id } = req.params;
+        const { body } = req;
+
+        res.json(user.update(id, body));
+    }
+
+    delete(req, res) {
+        const { id } = req.params;
+
+        res.json(user.delete(id));
```

Comment:
Such kind of response does not properly handles errors. Express returns status 500 and HTML with full stack trace, as well as in console there is a huge stack trace also. Could you please make correct error handling for all routes, not only for get?

Reply:
done